homeassistant/components/modbus/services.yaml

```diff
@@ -14,13 +14,21 @@ write_coil:
       name: State
       description: State to write.
       required: true
-      example: false
+      example: "0 or [1,0]"
       selector:
         object:
     unit:
```

Comment:
It's weird that we have 2 things that do the same. Should we undocument one, so it's not confusing users?

Reply:
Good idea, done.